nl.json

```diff
@@ -72,19 +72,19 @@
             "home" : "home",
             "shop" : "winkel",
             "cosmetics" : {
-                "dropdown" : "cosmetica",
-                "list" : "list &amp; zoeken",
+                "dropdown" : "cosmetisch",
+                "list" : "lijst &amp; zoeken",
                 "reminders" : "herinneringen",
-                "history" : "shop history",
-                "upcoming" : "aankomend",
+                "history" : "winkel geschiedenis",
+                "upcoming" : "wat komt er aan",
                 "sets" : "item sets"
             },
-            "guideDropdown" : "gids",
+            "guideDropdown" : "guide",
             "news" : "in-game nieuws",
             "randomiser" : "willekeurige items",
             "about" : {
                 "dropdown" : "over",
-                "translate" : "vertalen",
+                "translate" : "vertalingen",
```

Comment:
English translation uses singular noun, so not sure why you'd change this to an plural noun.

Reply:
You are correct. My bad